README.md

```diff
@@ -178,6 +181,43 @@ are not sufficient, you can provide a custom GitHub token with the necessary per
     github-token: ${{ secrets.CUSTOM_GITHUB_TOKEN }}
 ```
 
+### UV_TOOL_DIR
+
+On Windows `UV_TOOL_DIR` is set to `uv-tool-dir` in the `TMP` dir (e.g. `D:\a\_temp\uv-tool-dir`).
+On GitHub hosted runners this is on the much faster `D:` drive.
+
+On all other platforms the tool environments are placed in the
+[default location](https://docs.astral.sh/uv/concepts/tools/#tools-directory).
+
+If you want to change this behaviour (especially on self-hosted runners) you can use the `tool-dir`
+input:
+
+```yaml
+- name: Install the latest version of uv with a custom tool dir
+  uses: astral-sh/setup-uv@v3
+  with:
+    tool-dir: "/path/to/tool/dir"
```

Comment:
Should this not just be done via `env:`?

Reply:
A user would have to add this to every step using uv then. That's why I decided against it.

Reply:
Ah ok, smart